### PR TITLE
MAUI-3665-DateRestrictionUGContent_MAUI_SfCalendar

### PR DESCRIPTION
**Pull request by Priyadharshini-Periyasamy**
## *Description*

- Need to add the date restriction UG content for MAUI SfCalendar

### CODE REVIEW

File: MAUI/Calendar/date-restrictions.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+description: Learn here all about Date restrictions feature of Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Date Restrictions in MAUI Calendar (SfCalendar)
+
+## Minimum DateTime
+The [MinimumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_MinimumDate) property will restrict the backward navigation of date selection and also you cannot able to swipe beyound the minimum date range. The dates before the minimum date time is considered as disabled date.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-restrictions.md:13]: The [MinimumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_MinimumDate) property will restrict the backward navigation of date selection and also you cannot swipe beyond the minimum date range.

File: MAUI/Calendar/date-restrictions.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+  this.Calendar.MinimumDateTime = DateTime.Now.AddDays(-5);
+
+{% endhighlight %}
+{% endtabs %}
+
+![minimum-date-month-view-in-maui-calendar](images/date-restriction/minimum-date-month-view-in-maui-calendar.png)
+
+## Maximum DateTime
+The [MaximumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_MaximumDate) property will restrict the forward navigation of date selection and also you cannot able to swipe beyound the maximum date range. The dates after the maximum date time is considered as disabled date.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-restrictions.md:31]: The [MaximumDateTime](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_MaximumDate) property will restrict the forward navigation of date selection and also you cannot swipe beyond the maximum date range.

File: MAUI/Calendar/date-restrictions.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+this.Calendar.MaximumDateTime = DateTime.Now.AddDays(5);
+
+{% endhighlight %}
+{% endtabs %}
+
+![maximum-date-month-view-in-maui-calendar](images/date-restriction/maximum-date-month-view-in-maui-calendar.png)
+
+## Enable past dates
+To enable or disable the dates before the today date by using the [EnablePastDates](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_EnablePastDates) property of the `SfCalendar` and  you cannot able to select the dates before the today date while it is disabled.
```
Comment (laksmipriyamurugesan) [inline MAUI/Calendar/date-restrictions.md:49]: To enable or disable the dates before today's date using the [EnablePastDates](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.SfCalendar.html#Syncfusion_Maui_Calendar_SfCalendar_EnablePastDates) property of the `SfCalendar` and  you cannot select the dates before the today date while it is disabled.

File: MAUI/Calendar/date-restrictions.md (hunk 1 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-restrictions.md]: Comments modified

File: MAUI/Calendar/date-restrictions.md (hunk 2 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-restrictions.md]: Contents modified

File: MAUI/Calendar/date-restrictions.md (hunk 3 above)
Comment (Priyadharshini-Periyasamy) [reply MAUI/Calendar/date-restrictions.md]: Contents changed

**Review by Sasikumar3595 — APPROVED** [review]
This is fine

**Pull request merged**